.markdownlint.yaml

```diff
@@ -0,0 +1,13 @@
+default: false
+
+# Tags
+spelling: true
+indentation: false
+links: true
+whitespace: true
+html: true
+
+# Individual tweaks
+MD010:
+  code_blocks: false # Disallow hard tabs except in code blocks.
+MD028: false # Allow blank lines between blockquotes.
```

Comment:
I'm for dropping this one.

Reply:
You mean allowing this one?

Reply:
I mean that I think we should disallow blank lines between blockquotes (and fix the few that we have).

Reply:
Allowing this means you can't do:

> A block quote

> A consecutive block quote

There was a couple places where we are doing this and it seemed appropriate, but am happy to go with your recommendation.